packages/frontend/src/components/Claim/ClaimingClosed.tsx

```diff
@@ -0,0 +1,15 @@
+import { FormWrapper } from 'src/components/Form/Form'
+import styled from 'styled-components'
+
+export const ClaimingClosed = () => {
+  return (
+    <BidStartWrapper>
+      <h2>Claiming is closed ⌛️</h2>
+    </BidStartWrapper>
+  )
+}
+
+const BidStartWrapper = styled(FormWrapper)`
+  padding: 0 0 0 170px;
+  justify-content: center;
+`
```

Comment:
Now, this style component exists in `BidAwaiting`, `ResultsAwaiting` and `ClaimingClosed` components. The only difference is in `BidAwaiting` because padding there is `0 0 0 135px;`, should it be different than in other cases? If it can be changed to 170px then deduplicate these styles, if not let's change names to match it's purpose.